src/module_1/module_1_meteo_api.py

```diff
@@ -88,81 +111,96 @@ def clean_data(raw_data):
                 cleaned_data[key] = value
     return cleaned_data
 
+
 def process_data(data):
+    """
+    Reads a df and for each city calculates mean + std for each variable
+    """
+    calculated_df = data[["city", "time"]].copy()
+    for var in VARIABLES.split(","):
+        # For each climate var in the loop, if the name is in variable it's stored
+        idxs = [col for col in data.columns if col.startswith(var)]
+        # Save each variable with its corresponding mean and std (axis=1 per each row)
+        # We have decided to compute the mean of all models for one day
+        # e.g. mean of all precipitation_sum for Madrid on 1950-01-01
+        # We could have done the mean of all occurences during one year for one model
+        calculated_df[f"{var}_mean"] = data[idxs].mean(axis=1)
+        calculated_df[f"{var}_std"] = data[idxs].std(axis=1)
+    return calculated_df
+
+
+def plot_data(data):
```

Comment:
Si cogéis el código que subimos los profesores y lo poneis en vuestra PR no tiene mucha gracia. Además, este no es necesariamente el mejor código ni la mejor solución. La gracia del code review es que sea el vuestro para que os lo podamos revisar y aprendáis en el proceso.